gen/SendableChooser.yml

```diff
@@ -15,8 +15,12 @@ classes:
       GetSelected:
         # weirdness because return type
         cpp_code: |
-          [](frc::SendableChooser<T> * __that) {
-            return __that->GetSelected();
+          [](frc::SendableChooser<T> * __that) -> py::object {
+            auto v = __that->GetSelected();
+            if (!v) {
```

Comment:
What if the selection is False or 0?

Reply:
The operator bool is for checking if the handle is valid or not.